pom.xml

```diff
@@ -36,6 +36,8 @@
         <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
         <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or
         newer to run. -->
+        <revision>1</revision>
+        <changelist>999999-SNAPSHOT</changelist>
         <jenkins.version>2.375.4</jenkins.version>
         <hpi-plugin.version>3.42</hpi-plugin.version>
```

Comment:
```suggestion
```
This is unneeded and configured in the parent pom, but not relevant to enabling CD.